stdlib/Random/docs/src/index.md

```diff
@@ -18,9 +18,8 @@ The PRNGs (pseudorandom number generators) exported by the `Random` package are:
 Most functions related to random generation accept an optional `AbstractRNG` object as first argument.
 Some also accept dimension specifications `dims...` (which can also be given as a tuple) to generate
 arrays of random values.
-In a multi-threaded program, you should generally use different RNG objects from different threads
-or tasks in order to be thread-safe. However, the default RNG is thread-safe as of Julia 1.3
-(using a per-thread RNG up to version 1.6, and per-task thereafter).
+
+The default global RNG is thread-safe as of Julia 1.3 (using a per-thread RNG up to version 1.6, and per-task thereafter). The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races. That is, even if using the `Xoshiro`, the manually created RNG must be done per thread (the global RNG is thread-safe not via `Xoshiro itself, but by the details of the global RNG handling).
```

Comment:
I find this new wording too confusing.

The first point is that we do not talk about the "global RNG", at this point in the page we didn't define it. Nowadays we should only talk about the "default RNG" in the manual, which is the RNG used by default when no explicit RNG is passed.

Otherwise, it looks like all the information was already there. If you want to insist on your point, I would just add to the already existing paragraph something along the lines of:
> do note that while `Xoshiro` and the default RNG both implement the same RNG algorithm, the default RNG is thread-safe but `Xoshiro` is *not*.